integration/mesos/src/main/java/alluxio/mesos/AlluxioScheduler.java

```diff
@@ -214,7 +215,8 @@ public void resourceOffers(SchedulerDriver driver, List<Protos.Offer> offers) {
                             .addVariables(
                                 Protos.Environment.Variable.newBuilder()
                                     .setName("ALLUXIO_MESOS_SITE_PROPERTIES_CONTENT")
-                                    .setValue(Configuration.getSitePropertiesFile())
+                                    // TODO(andrew): set the site properties here
+                                    .setValue("")
```

Comment:
these are not regressions - setting the *content* variable to the *name* of the site-properties file wasn't doing anything anyway